test/extended/builds/s2i_root.go

```diff
@@ -37,7 +36,7 @@ var _ = g.Describe("[sig-builds][Feature:Builds] s2i build with a root user imag
 		Before(oc)
 		defer After(oc)
 
-		firstArgString := fmt.Sprintf("%s~https://github.com/sclorg/nodejs-ex", image.LocationFor("registry.ci.openshift.org/ocp/4.7:test-build-roots2i"))
+		firstArgString := fmt.Sprintf("%s~https://github.com/sclorg/nodejs-ex", "registry.ci.openshift.org/ocp/4.8:test-build-roots2i")
```

Comment:
Conversely the test with this image passed:

`

:  [sig-builds][Feature:Builds] s2i build with a root user image should  create a root build and pass with a privileged SCC  [Skipped:Disconnected] [Suite:openshift/conformance/parallel] | 1m14s
-- | --


`

It is safe to ask the admins to mirror this one, and then we can add the `image.LocationFor` back in, but use `4.8` instead of `4.7`

Reply:
reminder @alicerum you still need to add the `image.LocationFor` back in.

just keep the update from the 4.7 to 4.8 ref